rollup.config.js

```diff
@@ -18,14 +18,17 @@ const external = (id) => !id.startsWith('.') && !id.startsWith('/') && !id.inclu
 export default [
   {
     input,
-    output: { format: 'cjs', file: pkg.main },
+    output: { format: 'cjs', file: pkg.main, exports: 'default' },
     external,
     plugins: [babel(getBabelOptions({ useESModules: false }))],
   },
 
   {
-    input,
-    output: { format: 'esm', file: pkg.module },
+    input: './src/index.esm.js',
+    output: [
+      { format: 'esm', file: pkg.module },
+      { format: 'esm', file: pkg.exports['.'].import },
```

Comment:
why is this additional output needed?

Reply:
This is the `mjs` output above :)
At first I wanted to add a package.json file, but since `dist` is purged before every build, the package.json was being nuked along with it, so it was tricky to handle... using `mjs` was easier

Reply:
But why is the mjs file needed at all?

Reply:
Well, we either need `mjs` or `package.json` so `node` knows to load it as an es module... is `package.json` better?

Reply:
isn't this sufficient? https://github.com/i18next/i18next/pull/1681/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R6

Reply:
ahh you mean the package.json in the dist/esm folder

Reply:
Personally, I would do something like this while build: `echo '{\"type\":\"module\"}' > dist/esm/package.json`

Reply:
sure! let me update

Reply:
done

Reply:
ah wait

Reply:
now it looks good :)